components/common/GraphiQL/GraphiQL.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { createGraphiQLFetcher, Fetcher } from '@graphiql/toolkit';
+import { GraphiQL as GraphiQLPlayground, GraphiQLProps } from 'graphiql';
+import 'graphiql/graphiql.min.css';
+
+export interface IGraphiQL extends GraphiQLProps {
+  bearToken?: string;
+  url: string;
+}
+
+export const GraphiQL: React.FC<IGraphiQL> = ({ bearToken, url, fetcher, defaultQuery }) => {
+  const headers = {
+    'content-type': 'application/json',
+  };
+
+  const sortedHeaders = bearToken ? { ...headers, Authorization: `Bearer ${bearToken}` } : headers;
```

Comment:
[suggestion] can combine `headers` and `sortedHeaders`,  use empty string as default Authorization should be alright

```
Authorization = bearToken ? `Bearer ${bearToken}` : '';
```

Reply:
Hello @HZHAIX 
This would lead to a warning in console. I preferred a sorted `headers`.